File: c_program/upper_lower_number.c
#include<stdio.h>
int main(){
  char ch;
  scanf("%c",&ch);
  if(ch>=65 && ch<=90){
    printf("Upper");
  }
  else if(ch>=97 && ch<=122){
    printf("lower");
  }
  else if(ch>=48 && ch<=57){
    printf("Number");
  }
  else{
    printf("symbol");
  }
  return 0;

}

File: python/upper_lower_number.py
ch = input("Enter a character:")
if(ord(ch)>=65 and ord(ch)<=90):
  print("Upper")
elif(ord(ch)>=97 and ord(ch)<=122):
  print("Lower")
elif(ord(ch)>=48 and ord(ch)<=57):
  print("Number")
else:
  print("Symbol")
